urllib3/contrib/appengine.py

```diff
@@ -177,7 +216,7 @@ def _get_absolute_timeout(self, timeout):
         if timeout is Timeout.DEFAULT_TIMEOUT:
             return 5  # 5s is the default timeout for URLFetch.
         if isinstance(timeout, Timeout):
-            if timeout.read is not timeout.connect:
+            if timeout._read is not timeout._connect:
```

Comment:
The change is to...make things work. :) There is no "read" or "connect" at all in urllib3.util.timeout. Though there is _read and _connect (object properties), and read_timeout and connect_timeout (@properties). https://github.com/kennethreitz/requests/blob/master/requests/packages/urllib3/util/timeout.py